utils/s2n_fork_detection.c

```diff
@@ -53,6 +53,11 @@ typedef __darwin_pthread_once_t pthread_once_t;
     #define MADV_WIPEONFORK 18
 #endif
 
+/* Sometimes (for example, on FreeBSD) MAP_INHERIT_ZERO is called INHERIT_ZERO */
+#if !defined(MAP_INHERIT_ZERO) && defined(INHERIT_ZERO)
```

Comment:
Okay, this is one way of doing it. But it breaks a bit if there is a third `*BSD` calling it `INHERIT_ZERO_MAP` (or whatever different name)... But I'm not sure what the prettiest way of handling this is anyway...

Reply:
I don't think there can be a pretty way to handle all possible names. We would just have to add another check for a third name. The best we can do is decide on one name our code uses (MAP_INHERIT_ZERO) and translate all other names to that so that later code can just assume MAP_INHERIT_ZERO.